include/battle.h

```diff
@@ -15,6 +15,7 @@
 #include "pokeball.h"
 #include "battle_debug.h"
 #include "battle_dynamax.h"
+#include "random.h" // for rng_value_t
```

Comment:
Did you have a particular reason for adding these `// for rng_value_t` comments to the includes?

Reply:
Not a very justifiable one.